panoramix/templates/panoramix/dashboard.html

```diff
@@ -66,7 +65,7 @@ <h4 class="modal-title" id="myModalLabel">Dashboard CSS</h4>
       </div>
     </div>
 </div>
-<div class="gridster content_fluid">
+<div class="gridster content_fluid" style="visibility: hidden;">
```

Comment:
Well I had the `visibility:hidden` in `panoramix.css` but it would flicker, while the javascript `.gridster` call alters the css or classes the pre and post css changes would show for a fraction of a second. 

When the `visibility:hidden` is in `panoramix.css` it flickers, but not when it's inline... Let's land this so I can get this release out and we can revisit later. Also the `display:none` option might be better but the jquery `$.show()` would animate while there's already a lot going on so I went for this approach instead. 

I think the location of the js (head vs tail) might have something to do with it as well, maybe all of our `entry` scripts should be head since they contain css now. Anyhow, I'm merging and pushing a release as this works well and we can work out the details later.